docs/UserGuide.md

```diff
@@ -67,8 +65,8 @@ This project is based on the [AddressBook-Level3 project](https://se-education.o
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
-* If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
-</box>
+* If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when co
```

Comment:
For this should it be copied over to the application. instead of just co?

Reply:
whoops ok i fixed it